server/src/game/player.js

```diff
@@ -1,11 +1,21 @@
 class Player {
+  /**
+   * @param {String} name 
+   * @param {Number} id 
+   * @param {String} role 
+   * @param {Number} position 
+   */
   constructor(name, id, role, position) {
     this.name = name;
     this.id = id;
     this.role = role;
     this.originalRole = role;
     this.position = position;
     this.hasActed = false;
+    this.asleep = false;
```

Comment:
this verbage could be confusing as in not awake during the night, or had the spell casted on them?

Reply:
That's a good catch! I renamed it to `actionDisabled`. Ya, that could be real confusing if we came back to this later.